packages/riverpod/lib/src/devtool.dart

```diff
@@ -1,7 +1,50 @@
-// ignore_for_file: public_member_api_docs
+// ignore_for_file: public_member_api_docs, invalid_use_of_visible_for_testing_member
 
 part of 'provider.dart';
 
+class RiverpodNode {
+  RiverpodNode({
+    required this.id,
+    required this.containerId,
+    this.name,
+    this.state,
+    required this.type,
+    required this.mightBeOutdated,
+  });
+
+  final String id;
+  final String containerId;
+  final String? name;
+  final Result<dynamic>? state;
```

Comment:
Perfect, I'll make the change.

Reply:
I can't change it to `Object` , the [element.getState()](https://github.com/rrousselGit/riverpod/pull/1471/files#diff-dde080f54ecba70a3a51f54fbb1046d3a66660e18b91c555209c38db474b3f7aR38) returns a dynamic, since it is a generic.